src/App.css

```diff
@@ -1,60 +1,167 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+
+html,
+body {
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    width: 100vw;
+    height: 100vh;
+}
 
 html {
-    font-family: 'Roboto', sans-serif;
+    box-sizing: border-box;
+    color: #333;
+    background-color: #fff;
+    font: 62.5% 'Open Sans', sans-serif;
 }
 
-.app {
-    margin: 0 20%;
+body {
+    font-size: 1.5rem;
 }
 
-.title {
-    text-align: center;
-    color: #222;
+*,
+*:before,
+*:after {
+    background-color:inherit;
+    box-sizing: inherit;
+    font-weight: normal
 }
 
-.capitalize {
-    text-transform: capitalize;
+h1 {
+    margin: 2rem 0 1rem 2rem;
+    font-size: 2.6rem;
 }
 
-.strong {
-    font-weight: 500;
+#repo-link {
+    margin: 2rem 0 1rem 2rem;
+    font-size: 1.3rem;
 }
 
-h2,
-p {
-    margin: 0;
+#repo-link a {
```

Comment:
Normally it is not a good idea to use id for adding styles because it can make it harder later to override styles and it also has some other downsides: https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

Reply:
I made this intentionally, my reasoning is this:
`#repo-link` is special because it's the only one that will be completely hidden when using an iframe. If I was to change this to a list of links (repo, releases, issues...) I would probably use a `<ul>` instead and put it in plural but I would still like to treat it differently from all others.